backend/btrixcloud/storages.py

```diff
@@ -619,75 +595,78 @@ def organize_based_on_instance_number(
             wacz_log_streams: List[Iterator[dict]] = []
 
             for wacz_file in instance_list:
-                wacz_key = key + wacz_file.filename
-                cd_start, zip_file = sync_get_zip_file(client, bucket, wacz_key)
-
-                log_files = [
-                    f
-                    for f in zip_file.filelist
-                    if f.filename.startswith("logs/") and not f.is_dir()
-                ]
-                log_files.sort(key=lambda log_zipinfo: log_zipinfo.filename)
-
-                for log_zipinfo in log_files:
-                    wacz_log_streams.append(
-                        stream_log_lines(
-                            wacz_key, wacz_file.filename, cd_start, log_zipinfo
+                wacz_url = wacz_file.path
+                if wacz_url.startswith("/data"):
+                    wacz_url = f"http://browsertrix-cloud-frontend.default{wacz_url}"
```

Comment:
Let's put this in an env var in chart/templates/configmap, so at least its not hard-coded here. That way, can include the `{{ Release.namespace }}` if its different.